### ISSUE TITLE
JSON.stringify ([ undefined ] ) returns "[#unspecified]"

### ISSUE THREAD

**Issue by vprunet** [manuel-serrano/hop#51]
Hi,

Undefined values in arrays should be handled as null values.
See [https://www.ecma-international.org/ecma-262/5.1/#sec-15.12.3](url) , NOTE 5.

```
$ hop --repljs

> JSON.stringify( [undefined] );
[#unspecified]
```

**Comment by manuel-serrano** [comment]
Hi Vincent,

Ah yes you are correct. I have fixed that on 3.1.x and 3.2.x. Thanks for
your report.

-- 
Manuel

**Comment by manuel-serrano** [comment]
Ah yes. I had fixed that in the 3.2.x branch but I had forgotten to back port to 3.1.x. This is now done.
Sorry about that.

**Comment by vprunet** [comment]
It's all fine now.
Thank you
